packages/jasmine/BUILD.bazel

```diff
@@ -81,7 +84,7 @@ pkg_npm(
     ],
     build_file_content = "",
     substitutions = {
-        "//packages/jasmine:jasmine_runner.js": "//:node_modules/@bazel/jasmine/jasmine_runner.js",
+        "//packages/jasmine/private:jasmine_runner_test.bzl": "//@bazel/jasmine:private.bzl",
```

Comment:
```suggestion
        "//packages/jasmine/private:index.bzl": "//@bazel/jasmine/private:index.bzl",
```

Reply:
actually the substitution is no longer needed, since we made the path the same and the pkg_npm has a common one for `//packages` -> `//@bazel`

Reply:
How sweet 👯